CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+## Cookiecutter Contribution vs Application Contribution
+
+Please note that this repository is only the cookiecutter and not the entire
+CDP tooling and infrastructure ecosystem. This repository ties all of our
+tooling together into a single easy to deploy and maintain repository.
```

Comment:
Nitpick: I would prefer "a single repository that's easy to deploy and maintain"